### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by achernigin1987** [onnx/tensorflow-onnx#27]
I'm trying to convert TF graph, containing ResizeNearestNeigbor (conversion implemented [here](https://github.com/onnx/tensorflow-onnx/pull/20)), but **segmentation fault** is raised.

```using tensorflow=1.8.0, onnx=1.2.1
2018-05-31 10:30:17.480932: I tensorflow/tools/graph_transforms/transform_graph.cc:264] Applying fold_batch_norms
2018-05-31 10:30:17.485136: I tensorflow/tools/graph_transforms/transform_graph.cc:264] Applying fold_old_batch_norms
2018-05-31 10:30:17.511056: W tensorflow/core/graph/graph_constructor.cc:582] Node 'IteratorGetNext' has 1 outputs but the _output_shapes attribute specifies shapes for 2 outputs. Output shapes may be inaccurate.
2018-05-31 10:30:17.529917: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2018-05-31 10:30:17.668859: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:898] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-05-31 10:30:17.669711: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1356] Found device 0 with properties: 
name: GeForce GTX 1080 major: 6 minor: 1 memoryClockRate(GHz): 1.8225
pciBusID: 0000:08:00.0
totalMemory: 7.92GiB freeMemory: 7.09GiB
2018-05-31 10:30:17.669730: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1435] Adding visible gpu devices: 0
2018-05-31 10:30:17.900738: I tensorflow/core/common_runtime/gpu/gpu_device.cc:923] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-05-31 10:30:17.900782: I tensorflow/core/common_runtime/gpu/gpu_device.cc:929]      0 
2018-05-31 10:30:17.900789: I tensorflow/core/common_runtime/gpu/gpu_device.cc:942] 0:   N 
2018-05-31 10:30:17.901037: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1053] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6849 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080, pci bus id: 0000:08:00.0, compute capability: 6.1)
tensorflow ops: Counter({'Const': 60, 'BiasAdd': 17, 'Conv2D': 17, 'Maximum': 16, 'Mul': 16, 'MaxPool': 5, 'ResizeNearestNeighbor': 5, 'ConcatV2': 5, 'Relu': 1, 'Placeholder': 1})
tensorflow attr: Counter({'T': 82, 'dtype': 61, 'value': 60, 'data_format': 39, 'padding': 22, 'strides': 22, 'use_cudnn_on_gpu': 17, 'dilations': 17, 'N': 5, 'ksize': 5, 'align_corners': 5, 'Tidx': 5, 'shape': 1})
onnx mapped: Counter({'Const': 60, 'BiasAdd': 17, 'Conv2D': 17, 'Maximum': 16, 'Mul': 16, 'MaxPool': 5, 'ResizeNearestNeighbor': 5, 'ConcatV2': 5, 'Relu': 1, 'Placeholder': 1})
onnx unmapped: Counter()
Segmentation fault
```

**Comment by guschmue** [comment]
does work on my 16.04 box. Is it possible for you to capture a stack trace?

**Comment by achernigin1987** [comment]
I've run model using gdb:
```
Thread 1 "python" received signal SIGSEGV, Segmentation fault.
0x00007ffff3d67c59 in ONNX_NAMESPACE::graphProtoToGraph(ONNX_NAMESPACE::GraphProto const&, bool) ()
   from /home/achernigin/tensorflow/lib/python3.5/site-packages/onnx/onnx_cpp2py_export.cpython-35m-x86_64-linux-gnu.so
```

**Comment by ram1897** [comment]
Is the issue Resolved? I'm getting the same error while converting a custom model

**Comment by ram1897** [comment]
Mine got solved by giving the OP node as softmax:0 instead of softmax in the command

**Comment by kadeng** [comment]
I still see this with onnx 1.3.0.

**Comment by pengwa** [comment]
@kadeng  maybe share more detailed command line outputs including the command you run? what model are you converting?

**Comment by guschmue** [comment]
which linux version is this? Can you run ```python -c "import onnx; print(onnx.__version__)"``` just to make sure it is finding the right onnx.

**Comment by vsoorya** [comment]
This issue still exists.
Model converted: VGG-16 frozen model code used: https://github.com/machrisaa/tensorflow-vgg
Ubuntu 16.04
Onnx 1.3.0
onnx_tf  1.1.2

**Comment by guschmue** [comment]
Not seeing this on similar setup.
```python -c "import onnx; print(onnx.__version__)"``` finds onnx 1.3 ?

**Comment by Sanyorke-Han** [comment]
I met the same problem today when trying an example in "https://towardsdatascience.com/converting-a-simple-deep-learning-model-from-pytorch-to-tensorflow-b6b353351f5d", I checked that onnx version on my machine is 1.6.0.

**Comment by stasysp** [comment]
> I met the same problem today when trying an example in "https://towardsdatascience.com/converting-a-simple-deep-learning-model-from-pytorch-to-tensorflow-b6b353351f5d", I checked that onnx version on my machine is 1.6.0.

I've fixed it with versions onnx 1.5.0, onnx_tf 1.3.0, tf 1.14.0, torch 1.3.1. Be careful that if you install onnx_tf after onnx by pip, onnx upgrades to 1.6.0
